runner/usage.go

```diff
@@ -34,19 +19,14 @@ Configuration for this process is in the form of a YAML file with the following
 
 	workerConfig: arbitrary data which forms the basics of the config passed to the worker;
 		this will be merged with several other sources of configuration.
-`
-}
 
-// Load a configuration file
-func Load(filename string) (*RunnerConfig, error) {
-	data, err := ioutil.ReadFile(filename)
-	if err != nil {
-		return nil, err
-	}
-	var runnercfg RunnerConfig
-	err = yaml.Unmarshal(data, &runnercfg)
-	if err != nil {
-		return nil, err
-	}
-	return &runnercfg, nil
+	getSecrets: if true (the default), then configuration is fetched from the secrets service
```

Comment:
Can't we just supply JSON/YAML defaults via tags vs. making this a pointer, defaulting nil, and treating as true if nil?

Reply:
golang says you shouldn't want to do that, so it's not supported :)

That said, I could apply the default in LoadRunnerConfig.